pkg/ifreload/ifreload.go

```diff
@@ -58,7 +59,8 @@ func (i *ifReload) ifreloadImage() string {
 func (i *ifReload) ensureNamespace(ns string) error {
 	_, err := i.cli.CoreV1().Namespaces().Create(&v1.Namespace{
```

Comment:
the Create call here means that you'll be fixing this problem on *new* clusters, but not on existing ones on upgrade -- on existing ones the annotation won't get added because there's no Update call.  Need to refactor ensureNamespaces such that if the create fails due to IsAlreadyExists, you then call Update.

Reply:
copy the pattern in (for example) applyConfigMap that uses RetryOnConflict.